.tugboat/config.yml

```diff
@@ -0,0 +1,38 @@
+# https://docs.tugboatqa.com/reference/tugboat-configuration/
+# https://docs.tugboatqa.com/starter-configs/tutorials/drupal-10/
+services:
+  webserver:
+    image: tugboatqa/php:8.3-apache
+    default: true
+    commands:
+      init:
+        - docker-php-ext-install opcache
+        - a2enmod headers rewrite
+        - echo "memory_limit = -1" >> /usr/local/etc/php/conf.d/my-php.ini
+        - ln -snf "${TUGBOAT_ROOT}/web" "${DOCROOT}"
+      build:
+        - composer install
+        - composer run drupal:install
```

Comment:
What if we used `drupal:install-dev` here, and added the `drush config:set` command to that script, so that we don't have to repeat ourselves?